lib/phoenix_live_view/helpers.ex

```diff
@@ -188,24 +188,31 @@ defmodule Phoenix.LiveView.Helpers do
   Renders a `Phoenix.LiveComponent` within a parent LiveView.
 
   While `LiveView`s can be nested, each LiveView starts its
-  own process. A LiveComponent provides similar functionality
-  to LiveView, except they run in the same process as the
-  `LiveView`, with its own encapsulated state.
+  own process. A LiveComponent provides functionality similar
+  to a LiveView, except it runs in the same process as the parent
+  `LiveView` and has its own encapsulated state.
 
-  LiveComponent comes in two shapes, stateful and stateless.
-  See `Phoenix.LiveComponent` for more information.
+  LiveComponent may be either stateful or stateless, see
+  `Phoenix.LiveComponent` for more information.
 
-  ## Examples
-
-  All of the `assigns` given are forwarded directly to the
-  `live_component`:
-
-      <%= live_component(MyApp.WeatherComponent, id: "thermostat", city: "Kraków") %>
+  The `assigns` are either received by the component's optional
+  `preload/2` if defined, or otherwise automatically merged to the
+  parent LiveView socket assigns after the component's `mount/1`
+  (therefore overriding existing assigns of the same name already in
+  the socket).
 
   Note the `:id` won't necessarily be used as the DOM ID.
```

Comment:
Yes, a bit messy, sorry.  I wanted to leave it just so that the "possibly expected" use of `id` was mentioned, regardless.  I'll have a look again; it should at least say "An `:id`...".